src/tool_getparam.c

```diff
@@ -1027,6 +1029,49 @@ static const struct LongShort *single(char letter)
   return singles[letter - ' '];
 }
 
+struct TOSEntry {
+  const char *name;
+  unsigned char value;
+};
+
+static const struct TOSEntry tos_entries[]= {
```

Comment:
Not that it really matters, but TOS/DSCP constants are also available from `#include <netinet/ip.h>` if you want to use them.

There you should also find the constants IPTOS_DSCP_VA (0x0b, RFC5865 Voice Admit) and IPTOS_DSCP_LE (0x04, RFC8622 Lower Effort) defined, which I don't see represented in this table. The full list is https://www.iana.org/assignments/dscp-registry/dscp-registry.xhtml#dscp-registry-1.

Reply:
Thank you. I initially used the implementation from #7762, but then the tests failed on some platforms that don't have `netinet/ip.h`, so I ended up just hardcoding them.

Thanks for the list, I'll update it.